linetools/analysis/absline.py

```diff
@@ -5,29 +5,33 @@
 from __future__ import print_function, absolute_import, division, unicode_literals
 
 import numpy as np
-import os
+import pdb
 
 from astropy import units as u
 from astropy import constants as const
 
-# Atomcic constant
+
+# Atomic constant
 atom_cst = (const.m_e.cgs*const.c.cgs / (np.pi * 
     (const.e.esu**2).cgs)).to(u.AA*u.s/(u.km*u.cm**2))
 
 # Perform AODM on the line
 def aodm(spec,idata):
-    """  AODM calculation 
+    """  AODM calculation on an absorption line
     See Savage & Sembach 1991, ApJ, 379, 245
```

Comment:
Please a leave a blank line between the single line summary (here """ AODM calculation on an absorption line) and any extra text below.  This makes Sphinx happy.